packages/cli/README.md

```diff
@@ -290,3 +290,10 @@ The URL folder is a resource location equal to base URL up-to the last forward s
 type: array
 describe: List of Chrome command line options to pass on browser start. Can be used to disable CORS to scan protected page: --browserOptions disable-web-security
 ```
+
+-   keepUrlFragment: --keepUrlFragment
+
+```sh
+type: boolean
+describe: To keep the hash fragment in the URLs. If set to false, it will remove the hash fragment from URL. For example, http://www.example.com/#foo will be considered as http://www.example.com. Default value is false.
```

Comment:
This only applies when crawling. If http://www.example.com/#foo is passed in, that page (with the routing) will be scanned. That's the current behavior so it's a breaking change if we alter it. We might want to try to capture that in the documentation, but I'm not sure how we'd want to express it

Reply:
In the Readme documentation, new parameter is mentioned for Crawl and batch scan sections and not for single Url scan. The new parameter will not make any difference if it is passed or not for single Url scan.